### ISSUE TITLE
The problem of using being null when we have a custom type property that is in another proto file.

### ISSUE THREAD

**Issue by Rezakazemi890** [Rezakazemi890/Bulk-proto-to-C-sharp-class-converter#8]
using being null when we have a custom type property that is in another proto file.

**Comment by mrvahedi68** [comment]
Can you please put the reproduced sample?

**Comment by Rezakazemi890** [comment]
Yes i try to create sample and criteria of issue soon. @mrvahedi68

**Comment by Rezakazemi890** [comment]
An example for this is the attached proto sample. There is a property in the TEST proto that has a reference to a type (TEST2) that is not available in the TEST file. In this case, the mentioned problem arises.

```syntax = "proto3";
package TEST;
import "google/protobuf/timestamp.proto";
import "TEST2.proto";
option csharp_namespace = "TEST.TestPackage";

message TEST {
    google.protobuf.Timestamp Date = 1;
    string Str1 = 2;
    string Str2 = 3;
    int32 Id = 4;
}

message SECONDTEMP {
    TEST Test = 1;
}

message THIRDTEMP {
    repeated TEST2 Tests = 1;
    bool More = 2;
}

**Comment by Rezakazemi890** [comment]
Fixed in 81f4122ec285655025308579c6bd6b3cc0c9ff56
